SDWebImage.podspec

```diff
@@ -59,6 +59,6 @@ Pod::Spec.new do |s|
       'USER_HEADER_SEARCH_PATHS' => '$(inherited) $(SRCROOT)/libwebp/src'
     }
     webp.dependency 'SDWebImage/Core'
-    webp.dependency 'libwebp', '~> 0.5'
+    webp.dependency 'libwebp', '>= 0.5'
```

Comment:
... We can't do this IMO, it typically indicates no constrain about the version of `libwebp`. What if `libwebp` changed  APIs in the future? we'll need to fix.

Reply:
emmmmm....

A good fix, but Is this really useful ? libwebp now have only v1.0.2 version. So even we limit it with <=2.0, does not actually do anything.

If you care about something in the future, we will release SDWebImage 5.0.0 soon. Which the WebP dependency is `~> 1.0` and have the sem version to limit < 2.0

Reply:
If you care about something in the future, people who use SDWebImage with API-breaked libwebp (maybe 1 year later after libwebp become v2.0 and change the API we use), can still use a limit in the Podfile `pod 'libwebp', `~> 1.0``

And, we will release SDWebImage 5.0.0 soon. Which the WebP coder dependency is `~> 1.0` and have the sem version to limit < 2.0...

Reply:
We assume `Google` not do evil things, `libwebp` only may changes `API` in `2.x`, then `4.x` would break, and users issue to us, we tell them how to fix them, list the `libwebp` version explicity or upgrade to `5.x`, TBO, this is not good, from the view of project management, we need to prevent these things, in theory, my `PR` #2628 also needs to discuss, should we really trust `libwebp` would not happen very bad things like regressions until `2.0`?